### ISSUE TITLE
Unable to export O365User workload

### ISSUE THREAD

**Issue by PavolVonSekule** [microsoft/Microsoft365DSC#1275]
#### Details of the scenario you tried and the problem that is occurring
Export-M365DSCConfiguration -GlobalAdminAccount $Global:source `
                            -ComponentsToExtract @('O365User') `
                            -Path "C:\Users\user\OneDrive\Documents\projects\1\trash\dsc\clone" `
                            -FileName "AADUser.ps1" `
                            -Quiet
#### Verbose logs showing the problem
Export-M365DSCConfiguration -GlobalAdminAccount $Global:source `
                            -ComponentsToExtract @('O365User') `
                            -Path "C:\Users\user\OneDrive\Documents\projects\1\trash\dsc\clone" `
                            -FileName "AADUser.ps1" `
                            -Quiet | Out-Null
 
[1/1] Extracting [O365User]...
    |---[1/14] admin@M365x00000.onmicrosoft.com⌛ Export took {10 seconds}

PS C:\Windows\System32\WindowsPowerShell\v1.0> 
#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
# this is export
```

#### Version of the DSC module that was used ('dev' if using current dev branch)
Generated with Microsoft365DSC version 1.21.616.1

**Comment by 0xab-c0de** [comment]
Agree on that. First time detected in module version: ***1.21.526.2***

The M365DSC event log shows an error: 

```
Log Name:      M365DSC
Source:        MSFT_O365User
...
Description:
Cannot bind argument to parameter 'DSCBlock' because it is an empty string.
```

**Comment by NikCharlebois** [comment]
Fixed in https://github.com/microsoft/Microsoft365DSC/pull/1280 and will be released with 1.21.630.1 on Wednesday
